packages/mantine/src/theme/Theme.tsx

```diff
@@ -78,12 +78,12 @@ export const plasmaTheme: MantineThemeOverride = {
             },
         },
         Modal: {
-            styles: (theme, {size, fullScreen}: ModalProps) => ({
+            styles: (theme, {fullScreen}: ModalProps, {size}) => ({
```

Comment:
I think the second argument is of type `ModalRootStylesParams` and not `ModalProps`

https://github.com/mantinedev/mantine/blob/8ee2220e924faa094328ef7c6cddd5fb89f85ed5/src/mantine-core/src/Modal/ModalRoot/ModalRoot.styles.ts#L19

Reply:
ModalRootStyleParams does not seem to be exported. should I not put a type then ? ModalProps extends fullScreen at some point (that's why I chose ModalProps in the first place)

Reply:
You can omit the type then, it's better than the wrong type